### DISCUSSION TITLE
Error temp i range

### DISCUSSION THREAD

**Discussion by Fizkal72** [teamalgoritma/community#175] [Q&A] [answered]
Kalau seperti ini kenapa ya?

![temp](https://user-images.githubusercontent.com/107348513/177762905-4e3b42e9-35d4-4bff-8bdd-64b0f74c2ab4.PNG)
![Error i range](https://user-images.githubusercontent.com/107348513/177762993-81d7c7b0-d868-4d7a-90c0-b89c48161dec.png)

**Comment by onlyphantom** [comment, marked as answer]
Hello Fizkal, What is the URL / web page you are trying to scrape? :)

The error is due to a failure to find any element that matches your specification (`<div class="kol-kontent3-1">`) within `table`. If you tell us the URL, I can do a quick inspection for you!

**Reply by Fizkal72** [reply]
Ini seperti yg di contoh sih Kak...                    URL: https://pusatdata.kontan.co.id/makroekonomi/inflasi

**Reply by onlyphantom** [reply]
Hello Fizkal, there is no div with the `kol-kontent3-1` class on that page:

![image](https://user-images.githubusercontent.com/16984453/177776779-5399bf74-4f7f-4ccc-b8e5-ea5fc6544f65.png)

**Reply by Fizkal72** [reply]
Thanks, But actually it does when I try it on another cell code (not `temp`)

![div kol-konten3-1](https://user-images.githubusercontent.com/107348513/177780696-0a2fba7d-df41-4218-b128-1757357aa837.png)

Not sure why
